packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -1413,7 +1413,7 @@ class _RenderListTile extends RenderBox {
       }
     }
 
-    // This attempts to implement the redlines for the vertical position of the
```

Comment:
Actually "redline" is a term of art (see for example https://www.quora.com/What-is-redline-document-in-UI-designing) so it's the other "red lines" that should be fixed.

We appreciate the attention to detail!

Reply:
Ohh good to learn.

My pleasure Hans.